misc/tools/acrntrace/scripts/vmexit_analyze.py

```diff
@@ -89,44 +89,55 @@ def parse_trace_data(ifile):
 
     global TSC_BEGIN, TSC_END, TOTAL_NR_EXITS
     last_ev_id = ''
-    tsc_enter = 0
     tsc_exit = 0
-    tsc_last_exit_period = 0
 
     fd = open(ifile, 'rb')
 
+    # The duration of one vmexit is tsc_enter - tsc_exit
+    # Here we should find the first vmexit and ignore other entries on top of the first vmexit
     while True:
         try:
             line = fd.read(struct.calcsize(TRCREC))
             if not line:
                 break
             (tsc, event, d1, d2) = struct.unpack(TRCREC, line)
-
             event = event & 0xffffffffffff
 
-            if event == VM_ENTER:
-                if TSC_BEGIN == 0:
-                    TSC_BEGIN = tsc
-                    tsc_exit = tsc
-                    TOTAL_NR_EXITS = 0
+            if event != VM_EXIT:
+                continue
 
-                tsc_enter = tsc
-                TSC_END = tsc_enter
-                tsc_last_exit_period = tsc_enter - tsc_exit
+            # We found it and should seek back one line as we will read it in the following loop
```

Comment:
Does "it" refer to vmexit? If so, should we use vmexit?

Reply:
"it" refers to "first vmexit". I will update the PR to use "the first vmexit".

Reply:
@KaigeFu Great, thanks!